torchmultimodal/modules/layers/codebook.py

```diff
@@ -206,3 +206,8 @@ def forward(self, z: Tensor) -> CodebookOutput:
         quantized = self._postprocess(quantized_flat, permuted_shape)
 
         return CodebookOutput(encoded_flat, quantized_flat, codebook_indices, quantized)
+
+    def extra_repr(self):
```

Comment:
another cosmetic change, this just shows more details about the layer when you print any model that uses it. So instead of:
```
VQVAE(
  (encoder): Linear(in_features=2, out_features=2, bias=False)
  (decoder): Linear(in_features=2, out_features=2, bias=False)
  (codebook): Codebook()
)
```
It will add details to Codebook like any other standard nn layer: `Codebook(num_embeddings=4, embedding_dim=5)`